### PR TITLE
bgp minitest platform unsup checks

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
Now runs clean on N7k.

### CODE REVIEW

**Comment by chburket** [comment]
:+1:

File: tests/test_router_bgp.rb [hunk 1]
```diff
@@ -469,66 +469,52 @@ def test_default_disable_policy_batching
...
-                 'disable_policy_batching_ipv4 default value should be nil')
-    else
-      assert_equal(bgp.default_disable_policy_batching_ipv4,
-                   bgp.disable_policy_batching_ipv4,
-                   'disable_policy_batching_ipv4 default value should be empty')
-    end
+    bgp.disable_policy_batching_ipv4 = 'xx'
+    assert_equal('xx', bgp.disable_policy_batching_ipv4,
+                 "bgp disable_policy_batching_ipv4 should be set to 'xx'")
+    bgp.disable_policy_batching_ipv4 = \
```
Comment (mikewiebe) [inline tests/test_router_bgp.rb:489]: Here and below for `ipv6` your are explicitly setting the default value, but we should also have a test for the default value right after we create the new object.

File: tests/test_router_bgp.rb [hunk 2]
```diff
@@ -972,45 +958,48 @@ def test_default_maxas_limit
...
+      bgp.neighbor_down_fib_accelerate = true
+      assert(bgp.neighbor_down_fib_accelerate,
+             "vrf #{@vrf}: bgp neighbor_down_fib_accelerate "\
+             'should be enabled')
+      bgp.neighbor_down_fib_accelerate = false
+      refute(bgp.neighbor_down_fib_accelerate,
+             "vrf #{@vrf}: bgp neighbor_down_fib_accelerate "\
+             'should be disabled')
+
+      default = bgp.default_neighbor_down_fib_accelerate
```
Comment (mikewiebe) [inline tests/test_router_bgp.rb]: Again here, we need a default test right after creating the object.

Comment (chrisvanheuveln) [inline tests/test_router_bgp.rb]: nag, nag... :-)

**Pull request merged**